XRTK-Core/Packages/com.xrtk.core/Extensions/InteractionMappingsExtensions.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using XRTK.Definitions.Devices;
+using XRTK.Definitions.Utilities;
+using XRTK.Interfaces.InputSystem;
+using XRTK.Services;
+
+namespace XRTK.Extensions
+{
+    /// <summary>
+    /// Extensions for the InteractionSource class to add haptics and expose the renderable model.
```

Comment:
This summary doesn't match what's actually going on in this class

Reply:
Doh, copy / paste error